velox/common/encode/Base64.h

```diff
@@ -71,6 +71,11 @@ class Base64 {
       const std::pair<const char*, int32_t>& payload,
       std::string& outp);
 
+  /// Encodes the specified number of characters from the 'data' and writes the
+  /// result to the 'output'. The output must have enough space, e.g. as
+  /// returned by the calculateEncodedSize().
+  static void encode_url(const char* data, size_t size, char* output);
```

Comment:
naming: encode_url -> encodeUrl

I see that there are existing functions named encode_url. This was probably copy-pasted from somewhere. Let's add a TODO to rename these as well. Consider sending a separate PR for rename.

Reply:
Added a TODO. Will do this in the next PR once this got merged.